src/common/token.ts

```diff
@@ -167,14 +171,22 @@ export class ArweaveToken implements TokenTools<Transaction.default> {
   }
 
   public async submitTx({ tx }: { tx: Transaction.default }): Promise<void> {
-    const response = await this.arweave.transactions.post(tx).catch((err) => {
-      this.logger.error('Failed to post transaction...', { err });
-      return {
-        status: 500,
-        statusText:
-          err instanceof Error ? err.message : 'Failed to post Arweave Tx!',
-      };
-    });
+    let response:
+      | {
+          status: number;
+          statusText: string;
+          data: unknown;
+        }
+      | undefined = undefined;
```

Comment:
you could move this in the try block and the `response.status` check to avoid using a `let` we don't do anything with the response

Reply:
229f046